json.html.markdown

```diff
@@ -58,10 +59,20 @@ Drawbacks of JSON include lack of type definition and some sort of DTD.
 
   "alternative style": {
     "comment": "check this out!"
-  , "comma position": "doesn't matter - as long as it's before the next key, then it's valid"
+  , "comma position": "doesn't matter, if it's before the next key, it's valid"
   , "another comment": "how nice"
   },
 
-  "that was short": "And, you're done. You now know everything JSON has to offer."
+
+
+  "whitespace": "Does not matter.",
+
+
+
+  "that was short": "And done. You now know everything JSON has to offer."
 }
 ```
+
+## Further Reading
+
+* [JSON.org](http://json.org) All of JSON beautifully explained using flowchart-like graphics.
```

Comment:
Might be worth mentioning that those "graphics" are actually state transition diagrams